cmd/cli/types.go

```diff
@@ -50,12 +50,18 @@ type TxOutputExt struct {
 	Value  string `json:"value"`
 }
 
+type ResourceLimit struct {
+	Type  string `json:"type"`
+	Limit int64  `json:"limit"`
+}
+
 // InvokeRequest proto.InvokeRequest
 type InvokeRequest struct {
-	ModuleName   string            `json:"moduleName"`
-	ContractName string            `json:"contractName"`
-	MethodName   string            `json:"methodName"`
-	Args         map[string]string `json:"args"`
+	ModuleName    string            `json:"moduleName"`
+	ContractName  string            `json:"contractName"`
+	MethodName    string            `json:"methodName"`
+	Args          map[string]string `json:"args"`
+	ResouceLimits []ResourceLimit   `json:"resource_limits"`
```

Comment:
We now support multi-contracts calls. `ResouceLimits ` may need to share with other `InvokeRequest `.

Reply:
one method for multiple contracts? @qizheng09

Reply:
one method